stable_baselines3/common/logger.py

```diff
@@ -33,6 +34,26 @@ def __init__(self, frames: th.Tensor, fps: Union[float, int]):
         self.fps = fps
 
 
+class Figure(object):
+    """
+    Figure data class storing a matplotlib figure and whether to close the figure after logging it
+    """
+
+    def __init__(self, figure: plt.figure, close: bool):
+        self.figure = figure
+        self.close = close
+
+
+class Image(object):
+    """
+    Image data class storing an image and data format (gym envs normally use 'HWC')
+    """
+
+    def __init__(self, image: Union[th.Tensor, np.ndarray, str], dataformats: str):
```

Comment:
maybe a link to supported dataformats would be useful?

Reply:
Added.